phases/ephemeral/witx/typenames.witx

```diff
@@ -248,8 +248,10 @@
     $path_rename_target
     ;;; The right to invoke `path_filestat_get`.
     $path_filestat_get
-    ;;; The right to change a file's size (there is no `path_filestat_set_size`).
+    ;;; The right to change a file's size.
     ;;; If `path_open` is set, includes the right to invoke `path_open` with `oflags::trunc`.
+    ;;; Note: there is no `path_filestat_set_size`. This follows POSIX design, which only has
+    ;;; `ftruncate` and does not provide `ftruncateat`.
```

Comment:
I found this confusing to read because there clearly is a thing `path_filestat_set_size` called on the line below.  I guess you are referring to the fact that there is no addition separate function with this name?

Reply:
Yes, the intended meaning is that there is no function named `__wasi_path_filestat_set_size`.

Reply:
Could you change the text to add the word "function" here, so that it's clear you're talking about a function by that name (which indeed doesn't exist) rather than a right (which does)?

Reply:
Done!